bin/run_on_all_chains.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
```

Comment:
There should be no space between `#!` and the program

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3025/commits/ed6aedae41ba6d2b85e3ce113a356e640daeb2f8